src/store/index.js

```diff
@@ -2,16 +2,43 @@ import Vue from 'vue'
 import Vuex from 'vuex'
 import modules from './modules'
 import EpochChain from '@aeternity/aepp-sdk/es/chain/epoch'
+import createPersistedState from 'vuex-persistedstate'
 import { wrapActionsWithResolvedEpoch } from './utils'
 
 Vue.use(Vuex)
-
 const store = new Vuex.Store({
+  plugins: [
+    createPersistedState({
+      key: 'localNetwork',
+      paths: ['networkList']
```

Comment:
You should sync with local storage only networks that user has added manually, in another case you won't be able to change the list of predefined networks, because they will be overridden by the content of local storage.